<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>536</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>50</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="left" stretch="2,3,8">
      <item>
       <layout class="QVBoxLayout" name="left_1">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_1">
          <property name="text">
           <string>协议类型</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="left_1_1">
          <property name="editable">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>UDP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TCP服务器</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TCP客户端</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>本地IP地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="left_1_2">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>本地端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="left_1_3"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="left_2">
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>连接</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="left_3">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>发送设置</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="tab1_1">
              <item>
               <widget class="QCheckBox" name="s_check_1">
                <property name="text">
                 <string>启用文件数据源</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="s_check_2">
                <property name="text">
                 <string>发送完自动清空</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="s_check_3">
                <property name="text">
                 <string>按十六进制发送</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="s_check_4">
                <property name="text">
                 <string>数据源循环发送</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="tab1_2">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>间隔</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_3"/>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>毫秒</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="tab1_3">
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="text">
                 <string>载入数据</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>清空发送显示</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>接收设置</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QCheckBox" name="r_check_1">
                <property name="text">
                 <string>接收转向文件</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="r_check_2">
                <property name="text">
                 <string>显示接收日期</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="r_check_3">
                <property name="text">
                 <string>十六进制显示</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="r_check_4">
                <property name="text">
                 <string>暂停接收显示</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="r_check_5">
                <property name="text">
                 <string>群聊模式服务</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="r_check_6">
                <property name="text">
                 <string>群聊回显开关</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="tab_2_2">
              <item>
               <widget class="QPushButton" name="pushButton_5">
                <property name="text">
                 <string>保存数据</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_6">
                <property name="text">
                 <string>清空接受显示</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="right">
      <item>
       <layout class="QVBoxLayout" name="right_1">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>                       数据接收区</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTextBrowser" name="textreceive">
             <property name="html">
              <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="right_2" stretch="1,3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>连接对象</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="r_lineEdit_1"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="right_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>目标IP地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="r_lineEdit_2"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>目标端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="r_lineEdit_3"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="right_4">
        <item>
         <widget class="QPlainTextEdit" name="textsend">
          <property name="plainText">
           <string>AA DD CC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>191</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>发送</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>网络助手</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
